packages/material-ui-lab/src/Alert/Alert.js

```diff
@@ -0,0 +1,153 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { withStyles, lighten } from '@material-ui/core/styles';
+import Close from '../internal/svg-icons/Close';
+import { IconButton } from '@material-ui/core';
+import CheckCircle from '../internal/svg-icons/CheckCircle';
+import Warning from '../internal/svg-icons/Warning';
+import Error from '../internal/svg-icons/Error';
+import Info from '../internal/svg-icons/Info';
+import { green, orange, red } from '@material-ui/core/colors';
+
+export const styles = theme => ({
+  root: {
+    borderWidth: 1,
+    borderStyle: 'solid',
+    borderRadius: theme.shape.borderRadius,
+    position: 'relative',
+    display: 'flex',
+    alignItems: 'center',
+    padding: '1em 0',
```

Comment:
We haven't been using em unit in the codebase much (if none at all), what about using a multiple of 8, in pixels? (the underlying grid uses 4px).